src/engine/bank1c.asm

```diff
@@ -315,20 +355,114 @@ Func_70214: ; 70214 (1c:4214)
 	ld [de], a
 	pop bc
 	dec c
-	jr nz, .asm_70230
-.asm_7023e
+	jr nz, .loop_tiles
+
+.next_medal
 	pop af
 	rlca
 	pop hl
-	ld bc, $0024
+	ld bc, 3 * 3 * 4
 	add hl, bc
 	pop bc
 	dec c
-	jr nz, .asm_70227
+	jr nz, .loop_medals
 	ret
 
-Unknown_7024a: ; 7024a (1c:424a)
-	INCROM $7024a, $7036a
+border_medal_tile: MACRO
+	dw \1 ; VRAM address
+	db \2 ; tile
+	db \3 ; pal (?)
+ENDM
+
+.SGBBorderMedalTiles
+; GRASS_MEDAL
+	border_medal_tile v0Tiles1 + $182, $3f, $10
+	border_medal_tile v0Tiles1 + $184, $40, $10
+	border_medal_tile v0Tiles1 + $186, $41, $10
+	border_medal_tile v0Tiles1 + $1c2, $42, $10
+	border_medal_tile v0Tiles1 + $1c4, $43, $10
+	border_medal_tile v0Tiles1 + $1c6, $44, $10
+	border_medal_tile v0Tiles1 + $202, $45, $10
+	border_medal_tile v0Tiles1 + $204, $46, $10
+	border_medal_tile v0Tiles1 + $206, $47, $10
+
+; SCIENCE_MEDAL
+	border_medal_tile v0Tiles1 + $282, $48, $10
+	border_medal_tile v0Tiles1 + $284, $49, $10
+	border_medal_tile v0Tiles1 + $286, $4a, $10
+	border_medal_tile v0Tiles1 + $2c2, $4b, $10
+	border_medal_tile v0Tiles1 + $2c4, $4c, $10
+	border_medal_tile v0Tiles1 + $2c6, $4d, $10
+	border_medal_tile v0Tiles1 + $302, $4e, $10
+	border_medal_tile v0Tiles1 + $304, $4f, $10
+	border_medal_tile v0Tiles1 + $306, $50, $10
+
+; FIRE_MEDAL
+	border_medal_tile v0Tiles1 + $382, $51, $10
+	border_medal_tile v0Tiles1 + $384, $52, $10
+	border_medal_tile v0Tiles1 + $386, $53, $10
+	border_medal_tile v0Tiles1 + $3c2, $54, $10
+	border_medal_tile v0Tiles1 + $3c4, $55, $10
+	border_medal_tile v0Tiles1 + $3c6, $56, $10
+	border_medal_tile v0Tiles1 + $402, $57, $10
+	border_medal_tile v0Tiles1 + $404, $58, $10
+	border_medal_tile v0Tiles1 + $406, $59, $10
+
+; WATER_MEDAL
+	border_medal_tile v0Tiles1 + $482, $5a, $10
+	border_medal_tile v0Tiles1 + $484, $5b, $10
+	border_medal_tile v0Tiles1 + $486, $5c, $10
+	border_medal_tile v0Tiles1 + $4c2, $5d, $10
+	border_medal_tile v0Tiles1 + $4c4, $5e, $10
+	border_medal_tile v0Tiles1 + $4c6, $5f, $10
+	border_medal_tile v0Tiles1 + $502, $60, $10
+	border_medal_tile v0Tiles1 + $504, $61, $10
+	border_medal_tile v0Tiles1 + $506, $62, $10
+
+; LIGHTNING_MEDAL
+	border_medal_tile v0Tiles1 + $1b8, $63, $10
+	border_medal_tile v0Tiles1 + $1ba, $64, $10
+	border_medal_tile v0Tiles1 + $1bc, $65, $10
+	border_medal_tile v0Tiles1 + $1f8, $66, $10
+	border_medal_tile v0Tiles1 + $1fa, $67, $10
+	border_medal_tile v0Tiles1 + $1fc, $68, $10
+	border_medal_tile v0Tiles1 + $238, $69, $10
+	border_medal_tile v0Tiles1 + $23a, $6a, $10
+	border_medal_tile v0Tiles1 + $23c, $6b, $10
+
+; FIGHTING_MEDAL
```

Comment:
I'm assuming this comment needs updating now?

Reply:
Indeed so